### ISSUE TITLE
Current installation instructions result in an invalid hook call in the rendering library

### ISSUE THREAD

**Issue by fauna-brecht** [fauna-brecht/fauna-schema-migrate#9]
![image](https://user-images.githubusercontent.com/56540741/108509323-642d2e00-72bd-11eb-90ba-b146c8885c85.png)

When cloning the repository, this issue does not occur.

**Comment by ptpaterson** [comment]
I think the issue is in `ink-select-input`.

https://github.com/vadimdemedes/ink-select-input/blob/ae8ef4cd3315826e5c26bea0993d5272c7fc8b49/package.json#L35-L36

The peer dependency are not the same as other ink repos.

**Comment by ptpaterson** [comment]
I've submitted pull requests to update them.  But you are already using forks as dependencies.  You can update them in your forks, too?

**Comment by fauna-brecht** [comment]
@ptpaterson, were you able to reproduce it? I wasn't the person who found the issue but I've never seen it and still am not able to reproduce it.  I could bump these libraries but I'd like to test and make sure that it solve the issue. Do you have any idea how it's triggered? Maybe something related with the environment?

**Comment by ptpaterson** [comment]
Yep, I did nothing but `npm install fauna-schema-migrate` then `node_modules/.bin/fauna-schema-migrate run` and got the subject error.

I noticed the peer dependency warnings during the install.

**Comment by ptpaterson** [comment]
It's the mismatched react versions that is my best guess, though I do not know why you are not getting the error.

This repo requires React version 17.  The ink and ink-related repos require a mixture of `^16.8.2` and `>=16.8.2` as peer dependencies.

I am using npm 7, which is automatically installing peer dependencies, and I have no idea if that is what is exposing the issue.

**Comment by fauna-brecht** [comment]
Installing npm7 indeed introduces the problem. Thanks! :)

**Comment by fauna-brecht** [comment]
The issue is indeed caused by explicitly installing react 17 as advised by the ink readme. That results in react 17 to be installed while many ink-plugins depend on react 16 which caused the conflict. 
I have removed the explicit react 17 which seems to fix it for npm 7. I started pushing to a different scoped npm for testing purposes.


```npm install @fauna-databrecht/staging-fauna-schema-migrate@0.1.8-debug-npx2```
and 
```npx staging-fauna-schema-migrate run ```

Should now work in npm7.
I still have to verify whether it still works in older npm versions, I might have to explicitly specify react for those. I will look into that tomorrow. Else I'll add >=16.8.2
